StanwoodDebugger/Extensions/UserDefaults+Keys.swift

```diff
@@ -9,9 +9,11 @@ import Foundation
 
 extension UserDefaults {
     struct Keys {
-        static let hintShown = "showHint"
+        static let isHintShown = "isHintShown"
     }
```

Comment:
@zolomatok can be removed now that we use #function

Reply:
ohh, right!